packages/client/hmi-client/src/router/index.ts

```diff
@@ -7,12 +7,14 @@ import SimulationPlanPlayground from '@/views/SimulationPlanPlayground.vue';
 import TheiaView from '@/views/theia.vue';
 import DocumentView from '@/views/document.vue';
 import Simulation from '@/views/Simulation.vue';
+import ProjectView from '@/components/projects/Project.vue';
```

Comment:
For consistency with other main "views", can we move `components/projects/Project.vue` to `views/Project.vue`?

Reply:
Good catch